packages/calcite-components/src/components/action-menu/action-menu.tsx

```diff
@@ -95,7 +95,7 @@ export class ActionMenu implements LoadableComponent {
   openHandler(open: boolean): void {
     this.activeMenuItemIndex = this.open ? 0 : -1;
     if (this.menuButtonEl) {
-      this.menuButtonEl.active = open;
+      this.menuButtonEl.toggleAttribute(activeAttr, open);
```

Comment:
We typically rely on internal props for this instead of setting data attributes on the host. Could we make this consistent here or in a follow-up PR if pressed for time with the upcomin release?

Reply:
Yeah I thought about the internal prop but the internal prop wouldn't actually do anything on the action.

Thought about a class but didn't want to pollute action with a class incase users remove all classes

We should have a good best practice doc for this though